### PR TITLE
Tag events for member sites (2021-2022)

### PR DESCRIPTION
**Pull request by beatrizserrano**
Not sure if I've been very consistent @NickSto

### CODE REVIEW

File: content/events/2021-09-papercuts/index.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ external_url:
 gtn: false
 contact: Community
 tags: [ cofest, papercuts ]
+subsites: [eu,us]
```
Comment (bgruening) [inline content/events/2021-09-papercuts/index.md:14]: all?

File: content/events/2022-03-small-scale/index.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ external_url:
 gtn: false
 contact: "Hans-Rudolf Hotz"
 tags: 
+subsites: [eu,us]
```
Comment (bgruening) [inline content/events/2022-03-small-scale/index.md:14]: all?

File: content/events/2022-04-26-impact-data/index.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+title: "Generating software containers"
+date: '2022-04-26'
+days: 5
+tease: "Workshop at IMPaCT-Data"
+continent: EU
+location: "INB, Online, Barcelona, Spain"
+location_url: "https://inb-elixir.es/"
+gtn: false
+contact: "Björn Grüning"
+tags: [ ]
```
Comment (bgruening) [inline content/events/2022-04-26-impact-data/index.md:11]: container?

File: content/events/2020-eccb/index.md [hunk 4]
```diff
@@ -10,6 +10,7 @@ image: "eccb-2020-new-trends-logo.jpg"
 external_url: ""
 contact: "Björn Grüning, Anup Kumar, Alireza Khanteymoori, Fotis Psomopoulos"
 gtn: true
+subsites: [eu,us]
```
Comment (bgruening) [inline content/events/2020-eccb/index.md:13]: ECCB sounds like EU

File: content/events/2021-07-22-dev-roundtable/index.md [hunk 5]
```diff
@@ -14,6 +14,7 @@ links:
 - text: "Video"
   url: "https://youtu.be/n2uwDaU-L8s"
 tags: [ "devroundtable" ]
+subsites: [eu,us]
```
[the remark this answers is not in the document]
Comment (beatrizserrano) [reply content/events/2021-07-22-dev-roundtable/index.md:17]: I think that I did it wrong here, shouldn't have tagged at all

File: content/events/2021-07-sortee/index.md [hunk 6]
```diff
@@ -10,6 +10,7 @@ external_url: ""
 gtn: true
 contact: "Yvan Le Bras"
 image: sortee-logo.png
+subsites: [eu,us]
```
[the remark this answers is not in the document]
Comment (beatrizserrano) [reply content/events/2021-07-sortee/index.md:13]: no, wrong

File: content/events/2022-04-26-impact-data/index.md (hunk 3 above)
Comment (beatrizserrano) [reply content/events/2022-04-26-impact-data/index.md:11]: I think that the tags are defined in the config.json and useful to create dynamic pages, e.g. papercuts, roundtables, community calls...

File: content/events/2020-eccb/index.md (hunk 4 above)
Comment (beatrizserrano) [reply content/events/2020-eccb/index.md:13]: but it was online...

**Comment by NickSto** [comment]
Hi @beatrizserrano and @bgruening, I think we first need to determine what effect these tags will have. E.g. if you tag something as `eu`, which event feeds should it show up in besides the EU one? The global one? The GenOuest one?

I made an issue to discuss: #1317

**Comment by NickSto** [comment]
Okay I've implemented the new system in #1323. But in the process, I made #1322 which conflicts with this PR. But the new system means that most of the edits in this PR should probably be `[global, eu, us]` instead of `[eu, us]` anyway.

**Comment by beatrizserrano** [comment]
@NickSto I think that this is ok now, I've undone the tagging and the conflict seems to be solved

**Comment by NickSto** [comment]
Oh okay yeah so it's just a couple updates now? Sounds good.

**Pull request merged**